weblab/templates/entities/entity_version.html

```diff
@@ -63,10 +63,14 @@ <h2 id="entityversionname">
         {% endif %}
         <li><a href="{% entity_url 'version_list' entity %}" title="Show differences between versions of this {{ type }}">Other versions</a></li>
       </ul></div>
-      <!--
-      {% if type == 'model' %}
-        <a class="button" href="" title="Fit the parameters of this model to data">Fit</a>
+      {% if entity.entity_type == 'model' %}
+        <a class="button" href="{% url "fitting:result:new" %}?model={{entity.pk}}" title="Fit the parameters of this model to data">Fit</a>
```

Comment:
It would be nice to include the model version (id or sha) as an extra query parameter here (and similar for protocol & fitting spec below), so that it's filled in as the default in the form. Maybe allow the user still to change the version, but not the entity itself?